upstream-sha256: bbe93b2d1948bdcfc779ee422af578185709aca683f681de19a52b324f6c8b35
actual-sha256: 79a55b3e67d14e8b4bd4aa145bc667c0abfaa54e6347c59896e2471e3fe0dd71
--- upstream.expected
+++ tolk.actual
@@ -59,8 +59,8 @@
   hip_fp84 val39 = (*((hip_fp84*)((data1_2048+alu3))));
   int alu4 = ((alu0<<6)+alu1);
   float4 wmma0 = __WMMA_16_16_128_float8_e4m3_float(make_hip_fp832(val39.x,val39.y,val39.z,val39.w,val33.x,val33.y,val33.z,val33.w,val38.x,val38.y,val38.z,val38.w,val32.x,val32.y,val32.z,val32.w,val34.x,val34.y,val34.z,val34.w,val35.x,val35.y,val35.z,val35.w,val36.x,val36.y,val36.z,val36.w,val37.x,val37.y,val37.z,val37.w), make_hip_fp832(val31,val19,val26,val27,val28,val29,val30,val6,val17,val18,val20,val21,val22,val23,val24,val25,val0,val1,val2,val3,val4,val5,val7,val8,val9,val10,val11,val12,val13,val14,val15,val16), make_float4(0.0f,0.0f,0.0f,0.0f), 0, 0, 0, 0, 0, 0);
-  *(data0_256+(alu4+16)) = wmma0.y;
-  *(data0_256+(alu4+32)) = wmma0.z;
-  *(data0_256+(alu4+48)) = wmma0.w;
-  *(data0_256+alu4) = wmma0.x;
+  *(data0_256+(alu4+16)) = (0.0f+wmma0.y);
+  *(data0_256+(alu4+32)) = (0.0f+wmma0.z);
+  *(data0_256+(alu4+48)) = (0.0f+wmma0.w);
+  *(data0_256+alu4) = (0.0f+wmma0.x);
 }
